retrofit2-clients/src/main/java/com/palantir/remoting1/retrofit2/Retrofit2ClientBuilder.java

```diff
@@ -101,6 +102,11 @@ public Request authenticate(Route route, Response response) throws IOException {
             client.addInterceptor(new RetryInterceptor(config.maxNumRetries()));
         }
 
+        // HTTP request/response logging
+        HttpLoggingInterceptor httpLoggingInterceptor = new HttpLoggingInterceptor();
+        httpLoggingInterceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
```

Comment:
This is not OK as a default.
(1) Bodies can be huge, they shouldn't be logged in a default configuration
(2) Log-level should probably be TRACE, not INFO